modules/ROOT/pages/mule-upgrade-tool.adoc

```diff
@@ -0,0 +1,353 @@
+= Mule Upgrade Tool
+
+*Latest version: 1.0.0*
+
+include::release-notes::partial$mule-upgrade-tool/mule-upgrade-tool.adoc[tag=intro]
+
+The Mule upgrade tool runs in any environment that is capable of running a standalone Mule instance. This tool can update only standalone Mule instances that are running any Mule 4 version.
```

Comment:
Mule Runtime Upgrade tool provides a fully automated solution for patching and upgrading standalone on-premise installations of Mule Runtime, version 4.0 or above. The tool is available as a separate download and can be used with existing or new Mule Runtime standalone deployments. The tool is available as command line utility so it can be easily integrated with existing automated processes or workflows via scripts and cron jobs.